src/App.js

```diff
@@ -11,7 +11,9 @@ import LoginPage from './pages/LoginPage/LoginPage';
 import SignUpPage from './pages/SignUpPage/SignUp';
 import CartPage from './pages/CartPage/CartPage';
 import { useAuth } from "./context/AuthContext";
-import { ROUTE_PATH_CartPage, ROUTE_PATH_LandingPage, ROUTE_PATH_LoginPage, ROUTE_PATH_Mockman, ROUTE_PATH_ProductListPage, ROUTE_PATH_SignupPage, ROUTE_PATH_Unknown, ROUTE_PATH_WishListPage } from "./utils/Routes"
+import { ROUTE_PATH_CartPage, ROUTE_PATH_LandingPage, ROUTE_PATH_LoginPage,ROUTE_PATH_ProductContentPage, ROUTE_PATH_Mockman, ROUTE_PATH_ProductListPage, ROUTE_PATH_SignupPage, ROUTE_PATH_Unknown, ROUTE_PATH_WishListPage } from "./utils/Routes"
+import NotFoundPage404 from './pages/NotFoundPage404/NotFoundPage404';
+import ProductContentPage from './pages/ProductContentPage/ProductContentPage';
```

Comment:
Suggestion : create an "index.js" file to combine all the export for the pages at once. Then the imports will be easier

Reply:
I just need to import all pages to index and use it once in app.js rest of everything is just based on navigation, as in components index.js is good bcoz we reuse them